lib/manageiq/messaging/kafka/client.rb

```diff
@@ -69,6 +69,11 @@ def topics
           Rdkafka::Metadata.new(native_kafka).topics.collect { |topic| topic[:topic_name] }
         end
 
+        def metadata
+          native_kafka = producer.instance_variable_get(:@native_kafka)
+          Rdkafka::Admin.new(native_kafka).metadata
```

Comment:
You should be able to use `::Rdkafka::Config#admin` method here rather than having to use `instance_variable_get`, https://github.com/karafka/rdkafka-ruby/blob/main/lib/rdkafka/config.rb#L263-L286

```suggestion
          @kafka_client.admin.metadata
```